sympy/polys/matrices/domainmatrix.py

```diff
@@ -36,14 +100,13 @@ def from_rep(cls, ddm):
 
     @classmethod
     def from_list_sympy(cls, nrows, ncols, rows, **kwargs):
```

Comment:
You should add other sections like in the other methods like `Parameters`, `Returns` and `See Also` if required.

Reply:
Ok.